core/domain/skill_services.py

```diff
@@ -232,156 +232,161 @@ def get_all_skill_summaries():
     return skill_summaries
 
 
-def get_skill_summaries_in_batches(
-        num_queries_to_fetch, cursor=None, sort_by=None):
-    """Returns the summaries of all skills present in the datastore.
+def _get_skill_summaries_in_batches(
+        num_items_to_fetch, cursor, sort_by):
+    """Returns the summaries of skills present in the datastore.
 
     Args:
-        num_queries_to_fetch: number. Number of skills to fetch
-        cursor: str. The cursor to the next page.
+        num_items_to_fetch: int. Number of skills to fetch.
+        cursor: str or None. The cursor to the next page.
         sort_by: str. A string indicating how to sort the result.
 
     Returns:
-        4-tuple(skill_summaries, next_cursor, more, total_count). where:
-            skill_summaries: list(AugmentedSkillSummary). The list of
-                skill summaries.
-            next_cursor: str or None. A query cursor pointing to the
-                next batch of results. If there are no more results,
-                 this might be None.
+        3-tuple(skill_summaries, next_cursor, more). where:
```

Comment:
Explain more thoroughly how the number of returned results is determined, and what the cursor represents in the context of that number.

Reply:
Added more details. Done